test/go.mod

```
@@ -0,0 +1,8 @@
+module github.com/gruntwork-io/terraform-google-gke/test
+
+go 1.14
```

```suggestion
go 1.13
```
Use CircleCI infrastructure.